index.html

```diff
@@ -60,7 +56,248 @@ <h2 class="roboto">I'm a Software Developer</h2>
                     </ul>
                 </div>
             </div>
+        </section>
```

Comment:
A very beautiful project you here. Well done 👏🏾 🍾 

I see a few things that can further take you higher. 

- #### First Project Button
You currently have a colored border on this button. Following the Figma template, thiere should be not border. Consider this. 

Your Work | Figma Design
--- | ---
![image](https://user-images.githubusercontent.com/84156714/219877092-461a5329-0ccd-4799-add8-b4e790fb0436.png) | ![image](https://user-images.githubusercontent.com/84156714/219877162-a8fd5d31-7561-4095-9fef-2f1008a485b7.png)

- #### Commit message
Getting in the habit of creating quality commit messages makes using and collaborating with Git a lot easier. You will sure find very useful tips in [this article ](https://initialcommit.com/blog/git-commit-messages-best-practices) that will help you improve your commit messages.

Reply:
Thank you!! I appreciate your feedback, right now I'm going to eat that article !!